lavamoat/build-system/policy-override.json

```diff
@@ -14,6 +14,7 @@
     },
     "@eslint/eslintrc": {
       "packages": {
+        "<root>": true,
```

Comment:
[Q] What does this mean? Why was <root> added and is it a good thing?

Reply:
That doesn't sound right. LavaMoat isn't asking for access to anything, it has access already. All packages have access to everything by default, that's what LavaMoat is preventing. LavaMoat is _gating_ access, it's controlling which packages and APIs each package can access.

Here is the relevant portion of the quoted error message:

`package "@eslint/eslintrc" requested "<root>" as "/Users/elliot/code/metamask/metamask-extension/.eslintrc.base.js"`

Which means pretty much what it sounds like - the `@eslint/eslintrc` package is requesting the `<root>` package, but it wasn't explicitly granted access to that package so it was denied. The `<root>` package here _is not_ `@eslint/eslintrc`, it's the root _project_, the MetaMask extension itself.

I'm guessing this is the case because now the `.eslintrc.js` file is _extending_ another module in this repository. So when `@eslint/eslintrc` tries parsing that config file, it will have to import the base config as well, thus requiring access to the MetaMask extension itself. It didn't need that before, because we weren't extending from another config in this repo before. We were just extending from `@metamask/eslint-config` (which was already added to the policy override a few lines down from here, for this same reason).

Reply:
Ahh, okay. That makes more sense.